com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -169,6 +176,7 @@ public partial class UniversalRenderPipelineAsset : RenderPipelineAsset, ISerial
 
         // Quality settings
         [SerializeField] bool m_SupportsHDR = true;
+        [SerializeField] HDRFormat m_HDRFormat = HDRFormat.Default; // TODO: could just replace bool with the enum
```

Comment:
We could merge supports HDR to HDRFormat as false == None.
We could probably merge also SDR formats into this enum. Just have a ColorBuffer enum for SDR and HDR.
Thoughts?

Reply:
per light now.